src/components/Game/style.module.scss

```diff
@@ -0,0 +1,95 @@
+@import "../../styles/mixins.scss";
+@import "../../styles/variabels.scss";
+
+.header{
+    @include flexMixin(row,center,space-between);
+    align-self: stretch
+}
+
+.header__text {
+    @include ff-bold();
+    font-size:1.2rem;
+}
```

Comment:
move all nested blocks inside parent ones. For example, this part of the code will look like 


```
.header{
    @include flexMixin(row,center,space-between);
    align-self: stretch

    &__text {
        @include ff-bold();
        font-size:1.2rem;
    }
}
```